test/visual-diff/visual-tests

```diff
@@ -107,6 +107,27 @@
    *   // that a user would. (e.g., click on links, swipe gallery images.)
    *   "enable_percy_javascript": true,
    *
+   *   // [optional, EXPERIMENTAL] Path of JavaScript file relative to webserver
+   *   // root that exports a dictionary of (async) test functions keyed by a
+   *   // short descriptive name. Each test function is passed two arguments: a
+   *   // reference to the Puppeteer page, and the full name of the test (page
+   *   // name + test name). e.g., the JS file can be:
+   *   //   module.exports = {
```

Comment:
do we guarantee order execution here?

Reply:
No, but Percy doesn't guarantee order of snapshots either. Each test is run in a separate tab, so they aren't supposed to affect each other

Reply:
a that makes sense 👍